app/screens/planner/planner-screen.tsx

```diff
@@ -292,6 +303,13 @@ export const PlannerScreen = observer(function PlannerScreen({ navigation }: Pla
           value={formattedDate}
           style={{ marginBottom: spacing[5] }}
           onPress={() => setDatePickerVisibility(true)}
+          sideSection={
```

Comment:
Let's move this component out of the render function to a file of it's own. 
we need to start cleaning up the planner screen file :)